rpcs3/Loader/PSF.cpp

```diff
@@ -331,17 +328,17 @@ namespace psf
 			}
 			else if (fmt == format::string || fmt == format::array)
 			{
-				const std::string& value = entry.second.as_string();
-				const usz size = std::min<usz>(max, value.size());
+				std::string_view value = entry.second.as_string();
 
-				if (value.size() + (fmt == format::string) > max)
+				if (value.size() + u32{fmt == format::string} > max)
```

Comment:
can you please make all of these a bit more readable?
Like make a variable that tells us this is some zero terminator offset.
Right now it's incredibly weird to read and it was so before already.